Added example code/Notes and warnings (#751)
arduino/reference-en: Language/Functions/Bits and Bytes/bitRead.adoc

--
// OVERVIEW SECTION ENDS
=== Example Code
// Describe what the example code is all about and add relevant code
Reads the bit at the specified position and returns the value. In this example the binary representation of 6 is 0110, since `n=1` the value of the second bit from the right (which is 1 in this case) will be read and printed. 
[source,arduino]
----

> void setup() 
> {

Reviewer: ```suggestion
void setup() {
```
Arduino's standard code style uses attached braces.

>   Serial.begin(9600); 

Reviewer: ```suggestion
  Serial.begin(9600);
  while (!Serial) {
    ; // wait for serial port to connect. Needed for native USB port only
  }
```
This will make the example more friendly to users of the native USB boards (e.g., Leonardo).

> }
> 
> void loop() 
> {

Reviewer: ```suggestion

```
Let's put this in `setup()` so it only runs once.

>   int x = 6; int n = 1;       //setting the value of x and n

Reviewer: ```suggestion
  int x = 6;
  int n = 1;
```
This is more beginner friendly.

>   Serial.print(bitRead(x,n)); //reading the bit at position n and printing the value

Reviewer: ```suggestion
  Serial.print(bitRead(x, n));  //read the bit at position n and print the value
```
- Arduino style code formatting.
- Improve comment wording.

> }

Reviewer: ```suggestion
}

void loop() {
}
```
Necessary due to moving the code into `setup()`.

----
[%hardbreaks]

[float]
=== Notes and Warnings
The indexing of the rightmost bit starts from 0, so `n=1` reads the second bit from the right.

> Both `x` and `n` must be integer type.
> 

Reviewer: ```suggestion
```
I think this information would be better in the **Parameters** section, using the sample reference page as a model:
https://github.com/arduino/reference-en/blob/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc#parameters
